README.md

```diff
@@ -1,4 +1,4 @@
-MyExpenses
+MyExpenses  [![Bountysource](https://www.bountysource.com/badge/tracker?tracker_id=267118)](https://www.bountysource.com/trackers/267118-myexpenses?utm_source=267118&utm_medium=shield&utm_campaign=TRACKER_BADGE)
```

Comment:
In fact all the small buttons like these (there are a lot of this kind of buttons, for two others examples, see here https://github.com/f-droid/fdroidclient) are better next to the title, I think.

I can remove both if you want.